src/sentry/api/serializers/models/activity.py

```diff
@@ -1,7 +1,9 @@
 import functools
 
 from sentry.api.serializers import Serializer, register, serialize
+from sentry.api.serializers.models.commit import CommitWithReleaseSerializer
 from sentry.models import Activity, Commit, Group, PullRequest
+from sentry.utils.compat import zip
```

Comment:
We should be able to remove this import as well, this is just leftover from the python 2 -> 3 upgrade